packages/amplication-server/src/core/team/dto/WhereTeamAssignmentInput.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, InputType } from "@nestjs/graphql";
+
+@InputType({
+  isAbstract: true,
+})
+export class WhereTeamAssignmentInput {
```

Comment:
WhereTeamAssignmentInput sounds like you're trying to find a team assignment or so, but it's not receiving a team assignment id, so it's a bit confusing in first read. then i figured it's likely a prisma thing, and it really is after checking. so it now makes sense. leaving the comment here so you can see the line of thought.